src/content/docs/tutorial-create-alerts/manage-alert-quality.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: "Manage your alert quality"
+metaDescription: "How to manage the quality of your alerts with New Relic"
+redirects: 
+  - /docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/alert-quality-management-guide/
+  - /docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alert-quality-management-tutorial/
+  - /docs/new-relic-solutions/observability-maturity/aqm-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide
+---
+
+Teams suffer from alert fatigue when experiencing high alert volumes and alerts that aren't aligned to business impact. When this happens,incident responders often assume the alerts are false and have no business impact. In turn, they may start to prioritize easy-to-resolve alerts over others or close unresolved incidents to stay within their response time targets. This almost always results in slower incident response and increased scope and severity when major issues occur.
+
+Alert quality management (AQM) focuses on reducing the number of nuisance incidents so that you focus only on alerts with true business impact. This reduces alert fatigue and ensures that you and your team focus your attention on the right places at the right times.
+
+You're a good candidate for AQM if:
+
+* You have too many alerts.
+* You have alerts that stay open for long time periods.
+* You have a lot of alerts aren't relevant.
+* Your customers discover your issues before your monitoring tools do.
+
+<Callout variant="tip">
+Want to try a hands on learning approach before you start implementing this in your account? Check out the [alert quality management lab](https://learn.newrelic.com/hands-on-lab-alert-quality-management).
+</Callout>
+
+## Why use alert quality management? [#why-aqm]
+
+By using an alert strategy based on measuring business impact, you'll decrease response time and increase awareness of critical events. As you improve your alert signal to noise ratio, you'll reduce confusion and be able to rapidly identify and isolate the root cause of your problems. The overall goal of alert quality management is to lower the volume of less valuable alerts while providing easier ways to identify when more valuable incidents are created. This results in:
```

Comment:
`By using an alert strategy based on measuring business impact,`

Again, this language feels baroque

Reply:
Should `signal to noise` be hyphenated?

Reply:
` The overall goal of alert quality management is to lower...`

I don't think you need `alert quality management` here—the jargon adds to the word count w/o adding much clarity. Consider instead:

`The goal is to lower...`

Reply:
Also, look for places we can replace complex phrases with simpler verbs.

`lower the volume of` vs `reduce`